ydb/core/statistics/aggregator/tx_response_tablet_distribution.cpp

```diff
@@ -0,0 +1,98 @@
+#include "aggregator_impl.h"
+
+#include <ydb/core/protos/hive.pb.h>
+#include <ydb/core/statistics/stat_service.h>
+
+namespace NKikimr::NStat {
+
+struct TStatisticsAggregator::TTxResponseTabletDistribution : public TTxBase {
+    NKikimrHive::TEvResponseTabletDistribution Record;
+
+    bool SendAggregate = false;
+    bool ScheduleResolve = false;
+    bool ScheduleReqDistribution = false;
```

Comment:
NIT:
If I'm right only one of this bool can be true at the same time.
Maybe it will be more clear to make this enum
```cpp
enum class EAction : ui8 {
   None,
   SendAggregate,
   ScheduleResolve,
   ScheduleReqDistribution,
};
```